README.md

```diff
@@ -528,3 +528,9 @@ $ tree .
 For the client package, see https://godoc.org/github.com/theupdateframework/go-tuf/client.
 
 For the client CLI, see https://github.com/theupdateframework/go-tuf/tree/master/cmd/tuf-client.
+
+## Comparison to other implementations
+
+There are TUF implementations in a variety of programming languages. Some other Go implementations of TUF include:
+
+* [Notary](https://github.com/notaryproject/notary): A TUF implementation designed specifically for publishing and managing trusted collections of content. It is used by Docker Content Trust.
```

Comment:
Could you clarify:
1. How Notary v1 forked go-tuf?
2. How go-tuf has changed from Notary v1 since then?
Thanks!